Fix #2360: Remove unnecessary Initialisation of Firebase app in Testing

etigoel opened the pull request:
<!--
  - Thanks for submitting code to Oppia! Please fill out the following as part of
  - your pull request so we can review your code more easily.
  -->

## Explanation
Fix #2360 : Remove unnecessary Initialisation of Firebase app in Testing.
<!--
  - Explain what your PR does. If this PR fixes an existing bug, please include
  - "Fixes #bugnum:" in the explanation so that GitHub can auto-close the issue
  - when this PR is merged.
  -->

## Checklist
<!-- Please tick the relevant boxes by putting an "x" in them. -->
- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

etigoel committed:
Remove unnecessary Initialisation of Firebase app in Testing

FareesHussain commented:
@etigoel please check the checklist in the PR description

etigoel committed:
Remove unnecessary Initialisation of Firebase app in Testing

etigoel committed:
Remove unnecessary Initialisation of Firebase app in Testing

etigoel committed:
Merge branch 'remove-unnecessary-initialisation-of-firebase-app-in-testing' of https://github.com/etigoel/oppia-android into remove-unnecessary-initialisation-of-firebase-app-in-testing

etigoel committed:
Remove unnecessary Initialisation of Firebase app in Testing

etigoel commented:
> @etigoel please check the checklist in the PR description

Locally the tests are getting passed. Can we check if the tests are flaky.

FareesHussain commented:
> > @etigoel please check the checklist in the PR description
> 
> Locally the tests are getting passed. Can we check if the tests are flaky.
> 
> 

Add screenshots for the passing tests

etigoel committed:
Remove unnecessary Initialisation of Firebase app in Testing

rt4914 commented on .idea/misc.xml:
```diff
@@ -1,4 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
+  ?xml version="1.0" encoding="UTF-8"?>
```
Remove this line.

etigoel replied:
Done

anandwana001 commented on .idea/misc.xml:
```diff
@@ -1,4 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
+  ?xml version="1.0" encoding="UTF-8"?>
```
there should be no modification in this file.

anandwana001 reviewed (changes requested):
Thanks to @etigoel  for your first successful pull request. 

1. Please follow the checklist given in the Pull request description
      - PR title
      - PR description

2. Could you please confirm that after removing all the test cases are passing on Espresso? 
For Robolectric the test cases are passing on GitHub action so no need to check locally.

rt4914 reviewed (changes requested):
Suggested nit change.
Also, please follow the instructions mentioned in point 4. https://github.com/oppia/oppia-android/wiki#instructions-for-making-a-code-change

etigoel committed:
Fix #2360 Remove unnecessary Initialisation of Firebase app in Testing
Remove Firebase app initialisation as it has not been getting used.

etigoel committed:
Fix #2360 Remove unnecessary Initialisation of Firebase app in Testing
Remove Firebase app initialisation as it has not been getting used.

etigoel commented:
> Thanks to @etigoel for your first successful pull request.
> 
> 1. Please follow the checklist given in the Pull request description
>    
>    * PR title
>    * PR description
> 2. Could you please confirm that after removing all the test cases are passing on Espresso?
>    For Robolectric the test cases are passing on GitHub action so no need to check locally.

To run tests on espresso, I have created Android-instrumented-tests run configuration and its passing. Do we need to perform any other action.

etigoel committed:
Merge branch 'develop' into remove-unnecessary-initialisation-of-firebase-app-in-testing

etigoel commented:
> > > @etigoel please check the checklist in the PR description
> > 
> > 
> > Locally the tests are getting passed. Can we check if the tests are flaky.
> 
> Add screenshots for the passing tests

All the tests are passing currently.

etigoel commented:
The test is getting passed locally. 
![image](https://user-images.githubusercontent.com/66623641/103559739-a13f8b80-4edc-11eb-900f-aead8ccbc31d.png)

etigoel committed:
retrigger checks

etigoel committed:
Fix #2360 : Remove unnecessary Initialisation of Firebase app in Testing

This change has been done to remove unnecessary Initialisation of Firebase app.

rt4914 reviewed (approved):
LGTM, thanks.

rt4914 commented:
Will wait for @anandwana001 approval.

rt4914 commented:
@etigoel Please update title and description as per this comment: https://github.com/oppia/oppia-android/pull/2360#pullrequestreview-561023186

etigoel committed:
Merge branch 'remove-unnecessary-initialisation-of-firebase-app-in-testing' of https://github.com/etigoel/oppia-android into remove-unnecessary-initialisation-of-firebase-app-in-testing

etigoel committed:
Fix #2360 : Remove unnecessary Initialisation of Firebase app in Testing

This change has been done to remove unnecessary Initialisation of Firebase app.

etigoel committed:
Fix #2360 : Remove unnecessary Initialisation of Firebase app in Testing.

This change has been done to remove unnecessary Initialisation of Firebase app.

etigoel committed:
Fix #2360 : Remove unnecessary Initialisation of Firebase app in Testing.

This change has been done to remove unnecessary Initialisation of Firebase app.

etigoel committed:
Merge branch 'develop' into remove-unnecessary-initialisation-of-firebase-app-in-testing

anandwana001 reviewed (approved):
LGTM, Thanks @etigoel

rt4914 reviewed (changes requested):
Blocking because of title and description. 

As per mine and @anandwana001 previous comments please update the title and description of this PR so that you mention the issue number correctly.
Read this wiki and follow the 4th step to fix the PR. https://github.com/oppia/oppia-android/wiki#instructions-for-making-a-code-change

etigoel committed:
Fix #2360: Remove unnecessary Initialisation of Firebase app in Testing

Fix #2360 to remove unnecessary initialisation of Firebase app in Testing.

etigoel committed:
Fix #2360: Remove unnecessary Initialisation of Firebase app in Testing

Fix #2360 to remove unnecessary initialisation of Firebase app in Testing.

ShridharGoel commented:
@etigoel You also need to update the PR description to include the issue number.

etigoel committed:
Fix #2360: Remove unnecessary Initialisation of Firebase app in Testing.

Fix #2360 to remove unnecessary initialisation of Firebase app in Testing.

etigoel commented:
> @etigoel You also need to update the PR description to include the issue number.

For this I am using git commit -m "message" -m "message". Do we need to something else.

FareesHussain commented:
> > @etigoel You also need to update the PR description to include the issue number.
> 
> For this I am using git commit -m "message" -m "message". Do we need to something else.

@etigoel You need to do this in github go to this PR description and edit
![image](https://user-images.githubusercontent.com/54740946/103682302-b684ea80-4fae-11eb-87bb-08a8f6cb1412.png)

Luffy18346 commented:
Hi @etigoel, as @FareesHussain suggested above, you have to edit the PR and update the explanation.
For example, write "Fixes #(bug number): " (or "Fixes part of #(bug number)" if the PR only partially fixes an issue) below the explanation when you edit the PR. If you want, you can add further description about what issue this PR solves.

For the reference, you can take a look at 4th point in https://github.com/oppia/oppia-android/wiki#instructions-for-making-a-code-change

etigoel commented:
> Hi @etigoel, as @FareesHussain suggested above, you have to edit the PR and update the explanation.
> For example, write "Fixes #(bug number): " (or "Fixes part of #(bug number)" if the PR only partially fixes an issue) below the explanation when you edit the PR. If you want, you can add further description about what issue this PR solves.
> 
> For the reference, you can take a look at 4th point in https://github.com/oppia/oppia-android/wiki#instructions-for-making-a-code-change

Done.. Thank you

etigoel commented:
> > > @etigoel You also need to update the PR description to include the issue number.
> > 
> > 
> > For this I am using git commit -m "message" -m "message". Do we need to something else.
> 
> @etigoel You need to do this in github go to this PR description and edit
> ![image](https://user-images.githubusercontent.com/54740946/103682302-b684ea80-4fae-11eb-87bb-08a8f6cb1412.png)

Done.. Thanks

rt4914 reviewed (approved):
LGTM, thanks.
